easybuild/easyconfigs/r/Ruby/Ruby-2.1.5-foss-2015a.eb

```diff
@@ -0,0 +1,41 @@
+#    With <3 for EasyBuild
+#    
+#    EasyConfig for Ruby 2.1:
+#    ----------------------------------------------------------------------------
+#    Copyright: 2014 - Gregor Mendel Institute of Molecular Plant Biology GmBH
+#    License: MIT
+#    Authors: Aaron <aaron.zauner@gmi.oeaw.ac.at> Zauner
+#    ----------------------------------------------------------------------------
+
+name = 'Ruby'
+version = '2.1.5'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cache.ruby-lang.org/pub/ruby/']
+
+sanity_check_paths = {
+    'files': ['bin/ruby', 'bin/rake', 'bin/gem', 'bin/testrb', 'bin/erb',
+              'bin/ri', 'bin/irb', 'bin/rdoc', 'lib/libruby.so.2.1.0',
+              'lib/libruby-static.a', 'lib/libruby.so'],
+    'dirs': ['lib/ruby/2.1.0', 'lib/ruby/gems', 'lib/pkgconfig', 'include/ruby-2.1.0']
+}
+
+configopts = "--disable-install-doc --enable-shared"
+
+moduleclass = 'lang'
+
+
+exts_list = [
+    ('fpm', '1.3.3', {
```

Comment:
Shouldn't we try and have a stand-alone module for FPM?

Being able to include it as an extension is nice, but it would make more sense to have it readily available as a module itself, with Ruby as a dep?